### PR TITLE
Fix CI build?

### PR DESCRIPTION
**Pull request by paf31**
@garyb  Can you please help me figure out what's wrong here? I'm assuming having `psa` on the path would cause Pulp to run that instead of `psc`.

### CODE REVIEW

**Comment by garyb** [comment]
I'm not sure what's going on here, psa is usually enough to fix it. I guess it'll be fixed when pulp is released anyway...

File: package.json [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
   },
   "devDependencies": {
-    "jscs": "^2.8.0",
-    "jshint": "^2.9.1",
+    "eslint": "^3.19.0",
     "pulp": "^10.0.4",
-    "purescript-psa": "^0.3.9",
-    "rimraf": "^2.5.0"
+    "purescript": "^0.11.1",
+    "purescript-psa": "^0.5.0",
```
Comment (paf31) [inline package.json]: Wouldn't psa have to be installed globally so that it's on the path?

Comment (garyb) [inline package.json]: No, in fact I figured that might be the problem - `npm` scripts prefer the path of `.bin` in `node_modules` over the global path.

Comment (garyb) [inline package.json]: (Also I wasn't sure whether `pulp` checks it's neighbours for `psa` or whether it just tries it globally or something - the gulp or grunt plugin only used to use the npm-installed `purescript` when it was installed in the project)

**Pull request merged**